test/feature-benchmark/mzcompose.py

```diff
@@ -180,6 +174,40 @@ def run_one_scenario(
     return comparators
 
 
+def create_mz_image(
+    tag: str | None,
+    default_size: int,
+    additional_system_parameter_defaults: dict[str, str] | None,
+) -> Materialized:
+    return Materialized(
+        image=f"materialize/materialized:{tag}" if tag else None,
+        default_size=default_size,
+        # Avoid clashes with the Kafka sink progress topic across restarts
+        environment_id=f"local-az1-{uuid.uuid4()}-0",
+        soft_assertions=False,
+        additional_system_parameter_defaults=additional_system_parameter_defaults,
+        external_cockroach=True,
+        external_minio=True,
+        sanity_restart=False,
+    )
+
+
+def start_overridden_mz_and_cockroach(
+    c: Composition, mz: Materialized, instance: str
+) -> None:
+    with c.override(mz):
+        print(f"The version of the '{instance.upper()}' Mz instance is:")
+        c.run(
+            "materialized",
+            "-c",
+            "environmentd --version | grep environmentd",
+            entrypoint="bash",
+            rm=True,
+        )
+
+    c.up("cockroach", "materialized")
```

Comment:
This indent level needs to be increased, so that this `up` is under the `with`. This is vitally important in order to get the right Mz to start.

Reply:
Absolutely! Well spotted!
I had already similar issues during testing and this happens when moving code around (and the IDE "tries to help").

✅ Fixed.